src/frontend/components/UI/TextInputField/index.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { ChangeEvent, FocusEvent, ReactNode, useContext } from 'react'
-import classnames from 'classnames'
+import { default as cn } from 'classnames'
```

Comment:
A bit nitty-picky but normally we use this imported as `classNames` instead of `cn`.
Would be good to use the same one to make it easier to search in the future. Don't matter if `classNames`, `cn` or `cx`, but having the same its easier.

Reply:
sure thing. can't remember why I changed it